lib/src/typer.dart

```diff
@@ -12,11 +12,17 @@ class TyperAnimatedText extends AnimatedText {
   /// By default it is set to 40 milliseconds.
   final Duration speed;
 
+  /// The [Curve] of the rate of change of animation over time.
+  ///
+  /// By default it is set to Curves.ease.
+  final Curve curve;
```

Comment:
Is `ease` the best default value, or would `linear` be more consistent with current functionality?

See https://api.flutter.dev/flutter/animation/Curves-class.html

Reply:
Sure, that would seem nice. I shall set the default `curve` to `linear`.

I checked which is the default `curve` in `CurveTween`. Turns out it is `Curves.ease`: ![CurveTween](https://api.flutter.dev/flutter/animation/CurveTween-class.html).

Reply:
My thought is that the _old_ implementation was creating a `StepTween` between 0 and _n_.  Users upgrading should experience the _same_ behavior -- unless they explicitly specify a different `curve`.

Reply:
Agreed with @awhitford